wdfn-server/waterdata/templates/questions_comments.html

```diff
@@ -30,13 +30,18 @@ <h3 id="questions-comments-subtitle">Questions or Comments</h3>
         <form class="usa-form" action="{{ url_for('questions_comments', email_for_data_questions=email_for_data_questions) }}" method="post" >
             <fieldset id="feedback-destination-selection" class="usa-fieldset">
                 <legend class="usa-legend usa-legend">Let us help you to:</legend>
-                <div class="usa-radio">
-                    <input class="usa-radio__input" id="feedback-selection-contact" type="radio" name="feedback-type" value="contact" required>
-                    <label class="usa-radio__label" for="feedback-selection-contact">Contact someone about data at this location</label>
-                </div>
+
+                {% if page_type == "monitoring" %}
+                    <div class="usa-radio">
+                        <input class="usa-radio__input" id="feedback-selection-contact" type="radio" name="feedback-type" value="contact" required>
+                        <label class="usa-radio__label" for="feedback-selection-contact">Contact someone about data at this location</label>
+                    </div>
+                {% endif %}
                 <div class="usa-radio">
                     <input class="usa-radio__input" id="feedback-selection-report" type="radio" name="feedback-type" value="report">
-                    <label class="usa-radio__label" for="feedback-selection-report">Report a problem with this location</label>
+                    <label class="usa-radio__label" for="feedback-selection-report">
+                        Report a problem{%  if page_type ==  "monitoring" %} with this location {% endif %}
```

Comment:
Can we call the variable referring_page_type? I got a little confused here with the changes you made above that used request.endpoint. I think just changing the variable name helps.

Reply:
Sure can.